Examples/ArticleExample/ArticleExampleViewController.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  MediumViewController.swift
+//  CollectionKit
+//
+//  Created by yansong li on 2017-09-03.
+//  Copyright © 2017 lkzhao. All rights reserved.
+//
+
+import CollectionKit
+import UIKit
+
+class ArticleExampleViewController: UIViewController {
+  
+  var collectionView: CollectionView = {
+    let collectionView = CollectionView()
+    collectionView.backgroundColor = .white
+    return collectionView
+  }()
+
```

Comment:
Maybe instead of being a private function. we have use the let + block initialization for the articles that gets generated.
```swift


  let articles: [ArticleData] = {
    let count = 20
    return (1...count).map{
      ArticleData(hueValue: CGFloat($0) / CGFloat(count),
                  title: "Article \($0)",
        subTitle: "This is the subtitle for article \($0)")
    }
  }()
```

Reply:
Done